reactor-netty-core/src/main/java/reactor/netty/FutureMono.java

```diff
@@ -126,20 +113,41 @@ public void subscribe(CoreSubscriber<? super Void> s) {
 				return;
 			}
 
-			if (f.isDone()) {
```

Comment:
nit: remove the suppression above @SuppressWarnings("FutureReturnValueIgnored")

it is related to `addListener` invocation

Reply:
done